demos/common/cpp/models/src/detection_model_ssd.cpp

```diff
@@ -141,3 +193,25 @@ void ModelSSD::prepareInputsOutputs(InferenceEngine::CNNNetwork& cnnNetwork) {
     output->setPrecision(Precision::FP32);
     output->setLayout(Layout::NCHW);
 }
+
+void ModelSSD::prepareMultipleOutputs(OutputsDataMap& outputInfo) {
+    if (outputInfo.find("boxes") == outputInfo.end() && outputInfo.find("labels") != outputInfo.end()) {
+        throw std::logic_error("This model should have 'boxes' and 'labels' outputs");
+    }
+    outputsNames.push_back("boxes");
+    outputsNames.push_back("labels");
+
+    const SizeVector outputDims = outputInfo[outputsNames[0]]->getTensorDesc().getDims();
+    if (outputDims.size() != 2) {
+        throw std::logic_error("Incorrect output dimensions for Person Detection model");
+    }
+
+    maxProposalCount = outputDims[0];
+    objectSize = outputDims[1];
+    if (objectSize != 5) {
+        throw std::logic_error("Boxes output should have 5 as a last dimension");
+    }
```

Comment:
It would be better to unite error handling in one, with message:
`Incorrect shape of output layer containing boxes` 
or
`Incorrect shape of output layer containing boxes: (:, 5) expected, but got (x,..x)`

Reply:
Done, but in c++ style